Example/FinancialConnections Example/FinancialConnectionsUITests/XCUIApplication+Extensions.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  XCUIApplication+Extensions.swift
+//  FinancialConnections Example
+//
+//  Created by Krisjanis Gaidis on 4/26/23.
+//
+
+import Foundation
+import XCTest
+
+extension XCUIApplication {
```

Comment:
this was simpler than creating a new class (ex. `FinancialConnectionsApp` that abstracts `XCUIApplication`)